addons/stock/models/product.py

```diff
@@ -195,19 +195,23 @@ def get_components(self):
         self.ensure_one()
         return self.ids
 
+    def _get_description_or_name(self):
+        return self.description or self.name
+
     def _get_description(self, picking_type_id):
         """ return product receipt/delivery/picking description depending on
         picking type passed as argument.
         """
         self.ensure_one()
         picking_code = picking_type_id.code
-        description = self.description or self.name
+        description = self._get_description_or_name()
         if picking_code == 'incoming':
             return self.description_pickingin or description
         if picking_code == 'outgoing':
             return self.description_pickingout or self.name
         if picking_code == 'internal':
             return self.description_picking or description
+        return description
```

Comment:
Pretty sure you can achieve the same results with only this single line :wink:

Reply:
Yes, but as mrp_operation is added via mrp module, I though it's was good thing to handle it "properly" in mrp and to set a fallback in stock.
But of course, I can just keep this line if it's better

Reply:
Not sure to understand why https://github.com/odoo/odoo/pull/115309 is not necessary, it's the not really the same fix, as 15.0 manage HTML description

Reply:
Ok yes indeed, without the overriding way, it's the same fix

Commit and PR updated to reflect the review, other PR closed
Thanks